<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadWidget</class>
 <widget class="QWidget" name="LoadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="modeBtn">
     <property name="text">
      <string>PicMode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fileBtn">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
